src/OutputManager.php

```diff
@@ -81,20 +80,24 @@ private function setOutputDirectory()
      * Append a class to a file and save it
      * If no file is created the name of the class is the filename
      *
-     * @param PhpClass $class
+     * @param ZendClassGenerator $class
      */
-    private function saveClassToFile(PhpClass $class)
+    private function saveClassToFile($class)
     {
         if ($this->isValidClass($class)) {
-            $file = new PhpFile($class->getIdentifier());
+            $file = new FileGenerator();
+            $file->setFilename($class->getName() . '.php');
 
             $namespace = $this->config->get('namespaceName');
             if (!empty($namespace)) {
-                $file->addNamespace($namespace);
+                // This will _overwrite_ attached class namespace no _null_!
+                $file->setNamespace($this->config->get('namespaceName'));
```

Comment:
As I read Zend\Code [the class namespace will be output](https://github.com/zendframework/zend-code/blob/master/src/Generator/ClassGenerator.php#L925) [if we do not set a namespace for the file](https://github.com/zendframework/zend-code/blob/master/src/Generator/FileGenerator.php#L536).